packages/transaction-controller/src/TransactionController.ts

```diff
@@ -329,9 +342,12 @@ export class TransactionController extends BaseController<
     this.getNetworkState = getNetworkState;
     // @ts-expect-error TODO: Provider type alignment
     this.ethQuery = new EthQuery(provider);
+    this.isAdvancedGasFeeDisabled = disableAdvancedGasFee ?? false;
     this.isSendFlowHistoryDisabled = disableSendFlowHistory ?? false;
     this.isHistoryDisabled = disableHistory ?? false;
     this.registry = new MethodRegistry({ provider });
+    this.getAdvancedGasFeeByChainId =
+      getAdvancedGasFeeByChainId ?? (() => ({}));
```

Comment:
Is it more explicit if we return `undefined` for the default to make it clear there is no values, the same as if there were no saved values for a specific chain.

Reply:
Refactored as suggested